qupath-extension-processing/src/main/java/qupath/imagej/plugins/ImageJMacroRunner.java

```diff
@@ -209,7 +209,7 @@ public boolean runPlugin(final PluginRunner<BufferedImage> runner, final String
 
 
 
-	static void runMacro(final ParameterList params, final ImageData<BufferedImage> imageData, final ImageDisplay imageDisplay, final PathObject pathObject, final String macroText) {
+	public static void runMacro( final ParameterList params, final ImageData<BufferedImage> imageData, final ImageDisplay imageDisplay, final PathObject pathObject, final String macroText ) {
```

Comment:
This allows the ScriptableMacroRunner class to wrap the MacroRunner Class easily